apps/designer/app/routes/$.tsx

```diff
@@ -29,6 +30,11 @@ export const dynamicLinks: DynamicLinksFunction<CanvasData> = ({
 }) => {
   const searchParams = new URLSearchParams(location.search);
   searchParams.set("pageId", data.page.id);
+
+  // Break cache in case of css has changed
+  const cssHash = djb2a(JSON.stringify(data.tree));
```

Comment:
how much overhead would stringify be compared to having it upfront? if significant we could add that hash in project tree.ts loadById

Reply:
as in having the hash saved statically at publish time or so

Reply:
in this case we would be able to save both djb2a and stringify cost by simply doing this at publish time

Reply:
on the related note if we start publishing using a worker per project, CSS will become static, so hash can be made at publish time when we deploy the worker, so it would be done once anyways